code/segmentedtickerplant/datastripe.q

```diff
@@ -3,12 +3,17 @@
 //calls segmenting.csv which customer edits as to how many segments there will be
 //calls filtermap.csv which customer edits as to what filters should be applied to each segment
 
-//function transfoms segmenting.csv into table format so it can be accessed
-//function transforms filtermap.csv into table format then into a mapping of wcRef to filter which can be accessed and applied to data
+//configload transfoms segmenting.csv into table format so it can be accessed
+//configload transforms filtermap.csv into table format then into a mapping of wcRef to filter which can be accessed and applied to data
+
+//creating empty versions of segmenting table and filter mapping so we can revert to default mode if issues with either csv
+
+.stpps.segmentconfig:([] table:();segmentID:();wcRef:());
+.stpps.segmentfiltermap:(!/)([] wcRef:();filter:())`wcRef`filter;
 
 configload:{
-     .stpps.segmentconfig:("SIS";enlist",")0: hsym first .proc.getconfigfile["segmenting.csv"];
-     .stpps.segmentfiltermap:(!/) (("SS";enlist",")0: hsym first .proc.getconfigfile["filtermap.csv"])`wcRef`filter;
+     @[{.stpps.segmentconfig:{("SIS";enlist",")0: hsym first .proc.getconfigfile[x]}[x]};"segmenting.csv";{.lg.o[`stp1;"Failed to load segmenting.csv, revert to defaut mode and no segmenting."]}];
+     @[{.stpps.segmentfiltermap:{(!/)(("SS";enlist",")0: hsym first .proc.getconfigfile[x])`wcRef`filter}[x]};"filtermap.csv";{.lg.o[`stp1;"Failed to load filtermap.csv, revert to defaut mode and no segmenting."]}]; 
```

Comment:
Will need to change segmenting.csv and filtermap.csv to variables at somepoint. I'll approve for now so that adam can test it with his changes

Reply:
like josh said, these nfilename need to be variables, read in from command line. Also the logging shows that system will revert to default mode, but there is no logic to do this. I suppose this can be added in later but it will need to be noted.

The logging should use .lg.e instead of .lg.o, since its an error here. Also remove the reference to stp1, the proc name shouldnt be hard coded in here. What if this isnt the process name?